docs/development/contributing/code-of-conduct.rst

```diff
@@ -0,0 +1,26 @@
+.. _Code of Conduct:
```

Comment:
I'm happy with the Django/Speakup CoC.

We should acknowledge the licensing: https://creativecommons.org/licenses/by/3.0/